app/controllers/users_controller.rb

```diff
@@ -33,4 +33,30 @@ def load_user
   def user_params
     params.require(:user).permit(*USER_PERMITTED_PARAMS)
   end
+
+  def show
+    @user = User.find(params[:id])
+  end
```

Comment:
```suggestion
  def show; end
```
Đưa logic bên trong ra hàm load_user, xử lý thêm cho trường hợp k tìm thấy
Lưu ý dùng find_by thay vì find